intake_thredds/source.py

```diff
@@ -89,14 +89,21 @@ def _open_dataset(self):
                 else:
                     break
             path = self.path[i:]
+            if 'concat_dim' in self.xarray_kwargs:
```

Comment:
should we maybe allow more args from http://xarray.pydata.org/en/stable/generated/xarray.open_mfdataset.html?

Reply:
I like the idea of doing dataset concatenation. I think it would be beneficial not to expose the `concat_dim` option as a standalone argument. 👍🏽 for expanding the api to support other concat options. How about exposing concatenation options as `concat_kwargs` in `__init__`? I am imagining having `concat_kwargs=None` to be the default (which means the user doesn't want us to do any concatenation). With `concat_kwargs={....}`, we then go ahead and run `xr.concat(data, **concat_kwargs)`. 

What do you think?

Reply:
I see two ways here:
1.) try to implement xr.open_mfdataset API: http://xarray.pydata.org/en/stable/generated/xarray.open_mfdataset.html
2.) implement xr.concat(list, args) API: http://xarray.pydata.org/en/stable/generated/xarray.concat.html
but as I compare all concat_kwargs appear in open_mfdataset.

I like your implementation suggestion.

Reply:
so concat_dim=... and xarray_kwargs for all other kwargs of open_mfdataset